packages/ketcher-polymer-editor-react/src/components/rna/Switcher/Switcher.tsx

```diff
@@ -0,0 +1,119 @@
+import { Button } from '@mui/material'
+import styled from '@emotion/styled'
+import { ReactNode, useState } from 'react'
+import { Icon } from 'components/shared/ui/icon'
+import { IconNameType } from 'components/shared/ui/icon/icon'
+
+type SwitcherProps = {
+  selectedMonomers: [string, string, string]
+  setActiveMonomerType: (type: string) => void
+}
+
+type RAPButtonProps = {
+  isActive: boolean
+  callback: () => void
+  children: ReactNode
+}
+
+enum Monomers {
+  Nucleotide,
+  Nucleobase,
+  Sugar,
+  Phosphate
+}
+
+const svgNames: IconNameType[] = [
+  'rap-left-link',
+  'rap-middle-link',
+  'rap-right-link'
+]
+
+const RAPButton = ({ isActive, children, callback }: RAPButtonProps) => {
+  return (
+    <Button
+      css={({ color }) => ({
+        padding: '3px 12px',
+        backgroundColor: isActive
+          ? color.button.primary.active
+          : color.background.canvas,
+        borderRadius: '8px',
+        lineHeight: '18px',
+        minWidth: '33px',
+        color: isActive ? color.text.light : color.text.dark,
+        ':hover': {
+          backgroundColor: color.button.primary.hover,
+          color: color.text.light
+        }
+      })}
+      onClick={callback}
+    >
```

Comment:
It's up to you if you want to change it, but as long as we wanted to minimize inline `css` styles, you could write this component like this:

```tsx
...
import { useTheme, css } from '@emotion/react'
...
const RAPButton = ({ isActive, children, callback }: RAPButtonProps) => {

  const { color } = useTheme()

  const styles = css`
    padding: 3px 12px;
    background-color: ${isActive
      ? color.button.primary.active
      : color.background.canvas};
    border-radius: 8px;
    line-height: 18px;
    min-width: 33px;
    color: ${isActive ? color.text.light : color.text.dark};
    :hover {
      background-color: ${color.button.primary.hover};
      color: ${color.text.light};
    }
  `

  return (
    <Button css={styles} onClick={callback}>
      {children}
    </Button>
  )
}
```